package.json

```diff
@@ -20,7 +20,9 @@
     "build-rules": "node rules/build.js && cp rules/rules.json addon/",
     "version": "node update_version.js && git add addon/manifest.json",
     "vendor-copy": "mkdir -p addon/vendor && cp node_modules/mocha/mocha.* addon/vendor/ && cp node_modules/chai/chai.js addon/vendor/",
-    "prepublish": "npm run build-rules && npm run bundle"
+    "prepublish": "npm run build-rules && npm run bundle",
```

Comment:
Do you want to update the readme too? Just a short instruction on how to install it in FF